feature-about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/About.kt

```diff
@@ -255,13 +252,14 @@ private fun ExternalServiceImage(
 ) {
     Image(
         modifier = Modifier
-            .size(24.dp)
             .clickable {
                 navigateToExternalServices(
                     context = context,
                     serviceType = serviceType
                 )
-            },
+            }
+            .padding(12.dp)
+            .size(24.dp),
```

Comment:
This changes make buttons a little wider than the Figma design. However, I think that the ease of pressing the buttons is more important than the design, so I changed it.

Reply:
Agree. I told it to the designer. And I'm waiting the response🙏

Reply:
The designer says it is OK! Thank you for telling us!